src/Cron_Event_Command.php

```diff
@@ -243,10 +246,10 @@ public function run( $args, $assoc_args ) {
 				}
 			}
 			$events = $due_events;
-		} else if ( ! \WP_CLI\Utils\get_flag_value( $assoc_args, 'all' ) ) {
+		} elseif ( ! \WP_CLI\Utils\get_flag_value( $assoc_args, 'all' ) ) {
 			$due_events = array();
-			foreach( $events as $event ) {
-				if ( in_array( $event->hook, $args ) ) {
+			foreach ( $events as $event ) {
+				if ( in_array( $event->hook, $args, true ) ) {
```

Comment:
Just checking: will the variable types of `$event->hook` and the items in `$args` always be of the same type ?

Reply:
I think`$event->hook` will be contain string [ref](wp_schedule_event) `$hook`. Please do correct if wrong.